docs/get-started/scaffolding.md

```diff
@@ -0,0 +1,34 @@
+---
+id: scaffolding
+title: Scaffolding
+sidebar_label: Scaffolding
+---
+
+# Webiny Scaffold CLI
+
+### Introduction
+Sometimes when you develop applications with Webiny you will end up spending time doing common tasks that can easily become repetitive and tedious over time. This is where `webiny scaffold` comes into play.
+
+### Usage
+You will of course need to install Webiny's CLI if you haven't done so already: `npm -g install @webiny/cli`.
+
+You will be greeted with various templates to choose from:
```

Comment:
`After running "webiny scaffold", you will be prompted to choose from various templates. By default, Webiny comes with blah blaah templates, but you can create your own, or install the existing ones from NPM.`. Something like that, just to give people more context. And maybe create another guide `How to create scaffolding templates` and link here.

Reply:
@Pavel910 here you go